### ISSUE TITLE
Add starting date option for "Plays per Day" statistic 

### ISSUE THREAD

**Issue by sahinakkaya** [leepeuker/movary#576]
### Description

When I started using Movary, I added every movie I could remember to my watch history with their respective watch date as accurate as possible. The movie with oldest watch date is from 2008. This makes "Plays per Day" statistic unusable for me because it is always 0. 
![image](https://github.com/leepeuker/movary/assets/32161460/6e467ea3-79e0-47ea-b2be-af0cc888ce8b)
I started watching movies on a regular basis last, say, 6 months ago so I would prefer "Plays per Day" statistic to be calculated since that time. 

### Desired Behavior

Make an option to set starting date for "Plays per Day" statistic. Only movies watched after that date should be taken into account while calculating this statistic.

**Comment by sahinakkaya** [comment]
And yeah, this is very specific use case so I would totally understand if it doesn't make sense to implement this.

**Comment by leepeuker** [comment]
Hm, I see your point, but I think it would be weird if "Plays per day" would use a different time period than the rest of the statistics. I see to possible solutions here:
- set the time period for the statistics overall (e.g. only show stats for the past year), which is a feature I want to implement for a long time
- remove/swap the displayed statistics numbers, so that users can change the numbers they do not like or do not find relevant (but for this we need alternatives, we currently only have these six numbers to display)

**Comment by SirMartin** [comment]
I agree with everything except for the part you say is weird. I mean the only one that is hardly attached to the date is the plays per day. My case is more or less the same because I moved all my data from MovieDb, but I couldn't find a date for each play, so I just filled in random dates for all my old movies, and it was a thousand or something like that. But the rating, and the times I watched, I think like the case of @sahinakkaya is still accurate enough, like the amount of you replay a movie, or the total amount of movies. 

Of course, the option to have also stats by month/year etc, would be awesome.
